eth/beacon/helpers.py

```diff
@@ -402,6 +404,32 @@ def get_domain(fork_data: 'ForkData',
         slot,
     ) * 4294967296 + domain_type
 
+def verify_slashable_vote_data(state: BeaconState, vote_data: SlashableVoteData) -> bool:
+    vote_count = len(vote_data.aggregate_signature_poc_0_indices) + len(vote_data.aggregate_signature_poc_1_indices)
+    if vote_count > MAX_CASPER_VOTES:
+        return False
+
+    pubkeys = [
+        bls.aggregate_pubkeys([state.validators[i].pubkey for i in vote_data.aggregate_signature_poc_0_indices]),
+        bls.aggregate_pubkeys([state.validators[i].pubkey for i in vote_data.aggregate_signature_poc_1_indices])
+    ]
+
+    # TODO: change to hash_tree_root(vote_data) when we have SSZ tree hashing
+    vote_data_root = vote_data.root
+    messages = [
+        vote_data_root + bytes1(0),
+        vote_data_root + bytes1(1)
+    ]
+    return bls.verify_multiple(
+        pubkeys=pubkeys,
+        messages=messages,
+        signature=vote_data.aggregate_signature,
+        domain=get_domain(
+            state.fork_data,
+            state.slot,
+            DOMAIN_ATTESTATION,
```

Comment:
Needs to import `SignatureDomain`, which is inside `if TYPE_CHECKING:` currently. Change to:
```python
from eth.beacon.enums import (
    SignatureDomain,
)
```

And here we do:
```suggestion
            SignatureDomain.DOMAIN_ATTESTATION,
```

Reply:
awesome! i was actually right in the middle of adding it as a parameter to the function... like this:

```python
def verify_slashable_vote_data(state: BeaconState,
                               vote_data: 'SlashableVoteData',
                               max_casper_votes: int,
                               signature_domain_attestation: 'SignatureDomain') -> bool:
```

Which do we prefer?

Reply:
Oh, right right, we need to pass constants like `max_casper_votes` for testing. For example, we can set `shard_count` to 10 instead of 1024 for lightening the tests.

On the other hand, `SignatureDomain` is one of the enum constants that (i) we use as flags
(ii) similar the ID of the opcode, so I *think* it's unlikely to be changed in future forks.

So use:
```python
def verify_slashable_vote_data(state: BeaconState,
                               vote_data: 'SlashableVoteData',
                               max_casper_votes: int) -> bool:
```